src/widgets/Footer/Footer.tsx

```diff
@@ -12,22 +13,58 @@ import FooterComponent from './components/FooterComponent';
 import QuickLinksList from './components/QuickLinksList';
 
 export const Footer = () => {
-  return (
-    <div className="flex flex-col bg-secondary-10 px-5 pt-10 pb-4">
-      <h3 className="text-lg text-azaret font-medium mb-10 phone:hidden tablet:block">
-        LOGO
-      </h3>
+  const isMobile = useMediaQuery({ query: '(max-width: 768px)' });
```

Comment:
all our queries were already added. If you want to add a new breakpoint, you should make it in tailwind.config file. You can get rid of external libraries

Reply:
I believe that the use of the useMediaQuery hook here is justified by the fact that the footer in the mobile version and the desktop version have a very different configuration. And for nice code, it is better to use it. I can put the breaking point at 640x, but we don't have a tablet design

Reply:
Put it at 640px